src/Fixer/Alias/RandomApiMigrationFixer.php

```diff
@@ -162,7 +165,7 @@ protected function createConfigurationDefinition()
                 }])
                 ->setDefault([
                     'getrandmax' => 'mt_getrandmax',
-                    'rand' => 'mt_rand',
+                    'rand' => 'mt_rand', // TODO change to `random_int` as default on 4.0
```

Comment:
```suggestion
                    'rand' => 'mt_rand', // @TODO change to `random_int` as default on 4.0
```

so I can find it later